README.md

```diff
@@ -17,7 +17,7 @@ Run:
 ```bash
 mkdir build-linux
 cd build-linux
-cmake -D OEM_THEME_DIR=`pwd`/../nextcloudtheme ../client
+cmake -D OEM_THEME_DIR=$(realpath `pwd`/../nextcloudtheme)  ../client
```

Comment:
In that case the pwd command is not needed at all.

`cmake -D OEM_THEME_DIR=`realpath ../nextcloudtheme` ../client`

is enough